compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -479,8 +480,8 @@ class ComposeScene internal constructor(
         )
         needLayout = false
         forEachOwner { it.measureAndLayout() }
-        pointerPositionUpdater.beforeEvent(event)
-        processPointerInput(event)
+        pointerPositionUpdater.update()
```

Comment:
Here the difference with the previous code.

In case of relayout, we send always one additional move that is based on the last Move event, not on the current. Even if after that we send one more Move (that was passed to this function).

It was a missing event without which we can have some inconsistent behavior. Imagine a game that has a square that jumps in random position after we hover it:
```
move event 1 (300, 300) -> processPointerInput -> hover the square -> square receives Enter, and we set squarePosition = (750, 100)

move event 2 (750, 100) -> measureAndLayout, layout square to (750, 100) -> needUpdate() -> update -> send Move (300, 300) -> square recieve Exit -> ... processPointerInput -> 750, 100 -> square receives Enter.
```

The example is very synthtic thought, and in reality we don't need this. But besides that, I remember that Compose nodes don't work well if we send a different Move after the node changes its position (it is probably a bug of Compose node), so I would keep it just as a precaution.